src/control/cmd/dmg/pretty/network.go

```diff
@@ -32,6 +32,12 @@ import (
 	"github.com/daos-stack/daos/src/control/lib/txtfmt"
 )
 
+const (
+	providerTitle  = "Provider"
+	interfaceTitle = "Interfaces"
+	socketTitle    = "NUMA Socket"
+)
```

Comment:
Do these need to be package-level consts? Putting these out here means that these variable names aren't available for any other printers. Might not be a big deal but seems like it might get cluttered...

Reply:
Ok, I've made them locally defined strings.  Hopefully that's what you had in mind.